docs/quickstart.rst

```diff
@@ -418,8 +481,8 @@ required.
 Deploy the changes
 ==========================
 
-Deploy the changes as before and nothing should change because it generates exactly the same
-configuration.
+Deploy the changes as before, by connection to the servers terminal.
+Nothing will change because the generated configuration should be exactly the same.
 
 .. code-block:: sh
 
```

Comment:
Should line 495 link to another section? It does not for me, also not on docs.inmanta.com. You haven't changed it, I know, but we should set this straight.

Reply:
linked to a file that had it's name changed. resolved.